### PR TITLE
Verification lib api getter

### PR DESCRIPTION
**Pull request by leijerry888**
Verification Library API implementation of constructor and `get_library_item`

Let's make this PR as clean and comprehensive as possible and use it as API development example

Tag with DO NOT MERGE before we discuss it.

### CODE REVIEW

File: src/api/verification_library.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+def main():
+    vl_obj = VerificationLibrary("../../schema/library.json")
+    vl_obj.lib_items
+    print("testing")
+    print("dd")
+
+
+if __name__ == "__main__":
+    main()
```
Comment (lymereJ) [inline src/api/verification_library.py]: Are we just keeping that for development?

File: src/api/verification_library.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        }
+
+        return item_dict
+
+
+def main():
+    vl_obj = VerificationLibrary("../../schema/library.json")
+    vl_obj.lib_items
+    print("testing")
+    print("dd")
```
Comment (yunjoonjung-PNNL) [inline src/api/verification_library.py]: I believe this will be deleted in the future update.

**Review by yunjoonjung-PNNL — APPROVED** [review]
All looks great to me. Thanks for making a great example.

File: src/api/verification_library.py (hunk 1 above)
Comment (leijerry888) [reply src/api/verification_library.py]: I should have removed them. They are just there for quick debugging. Thanks for pointing it out!

File: src/api/verification_library.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                logging.warning(
+                    f"{len(shared_keys)} repeated verification item names identified when loading: {shared_keys}"
+                )
+
+            self.lib_items.update(v)
+            for lib_name, lib_content in v.items():
+                self.lib_items_json_path[lib_name] = k
+                if lib_name in globals().keys():
+                    python_path = inspect.getfile(
+                        globals()["AutomaticShutdown"].__class__
```
Comment (leijerry888) [inline src/api/verification_library.py:66]: bug found, fix needed

Comment (leijerry888) [reply src/api/verification_library.py:66]: already fixed in 70ac1af3

**Pull request merged**